internal/pkg/values/util.go

```diff
@@ -96,14 +96,16 @@ func GetLayerFromAnnotations(annotations map[string]string, logEvent resourceLog
 		}
 	}
 	if downscaleReplicas, ok := annotations[annotationDownscaleReplicas]; ok {
```

Comment:
```suggestion
	if _, ok := annotations[annotationDownscaleReplicas]; ok {
```

Reply:
Am I missing something here, the value is being used as an argument in strconv.ParseInt